Pages/Response.razor

```diff
@@ -0,0 +1,17 @@
+@page "/response"
+
+<h1>Response</h1>
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ChatGPT Response</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <textarea placeholder="ChatGPT response..."></textarea>
```

Comment:
I'm not sure if a `<textarea>` is the best way to do this? Maybe it is but the wireframes look more like it could be just a `<p>` tag, but at the very least, the textarea should have the `disabled` attribute so that users can't manually type something in to "edit" the response